pkg/envinfo/envinfo.go

```diff
@@ -44,7 +42,7 @@ type EnvInfoURL struct {
 	// Indicates if the URL should be a secure https one
 	Secure bool `yaml:"Secure,omitempty"`
 	// Cluster host
-	Host string `yaml:"host,omitempty"`
+	Host string `yaml:"Host,omitempty"`
```

Comment:
Does changing `host` to `Host` break anything (it shouldn't, but we should double check)

Reply:
I tested kube and it was fine, but i will double confirm it again